bin/main.ml

```diff
@@ -658,75 +658,87 @@ let check_path contexts =
             name
             (hint name (String.Set.to_list contexts))
 
-let resolve_targets ~log common (setup : Main.setup) user_targets =
+let resolve_path path ~(setup : Main.setup) =
+  let check_path = check_path setup.contexts in
```

Comment:
In both `resolve_path` and `resolve_target`, `check_path` is now used only once, so we don't need to partially apply it anymore